src/mainwindow.ui

```diff
@@ -44,50 +47,59 @@
     </item>
    </layout>
   </widget>
-  <widget class="QMenuBar" name="menubar">
-   <property name="geometry">
-    <rect>
-     <x>0</x>
-     <y>0</y>
-     <width>800</width>
-     <height>24</height>
-    </rect>
+  <widget class="QToolBar" name="toolBar">
+   <property name="font">
+    <font>
+     <family>Arial Nova Light</family>
+     <pointsize>10</pointsize>
+     <weight>75</weight>
+     <bold>true</bold>
+    </font>
    </property>
-  </widget>
-  <widget class="QToolBar" name="mode_toolbar">
    <property name="windowTitle">
-    <string>toolBar_2</string>
+    <string>toolBar</string>
    </property>
-   <property name="layoutDirection">
-    <enum>Qt::LeftToRight</enum>
+   <property name="toolButtonStyle">
+    <enum>Qt::ToolButtonTextBesideIcon</enum>
    </property>
-   <property name="movable">
+   <attribute name="toolBarArea">
+    <enum>TopToolBarArea</enum>
+   </attribute>
+   <attribute name="toolBarBreak">
     <bool>false</bool>
+   </attribute>
+   <addaction name="actionDashboard"/>
+   <addaction name="separator"/>
+   <addaction name="actionExplorer"/>
+  </widget>
+  <widget class="QToolBar" name="toolBar_2">
+   <property name="font">
+    <font>
+     <family>Arial Nova Light</family>
+    </font>
    </property>
-   <property name="orientation">
-    <enum>Qt::Vertical</enum>
+   <property name="windowTitle">
+    <string>toolBar_2</string>
    </property>
    <property name="iconSize">
     <size>
-     <width>24</width>
-     <height>24</height>
+     <width>26</width>
+     <height>26</height>
     </size>
    </property>
    <property name="toolButtonStyle">
-    <enum>Qt::ToolButtonIconOnly</enum>
-   </property>
-   <property name="floatable">
-    <bool>false</bool>
+    <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
    <attribute name="toolBarArea">
     <enum>LeftToolBarArea</enum>
    </attribute>
    <attribute name="toolBarBreak">
     <bool>false</bool>
    </attribute>
-   <addaction name="actionDashboard"/>
-   <addaction name="actionExplorer"/>
+   <addaction name="actionNew_connection"/>
    <addaction name="separator"/>
+   <addaction name="actionActual_state"/>
+   <addaction name="actionHistory"/>
```

Comment:
Are these two actions really connected to the PR? Maybe they should be in their own branch.